static_src/js/main.js

```diff
@@ -267,7 +272,37 @@ function initializeJS() {
 	})();
 })(jQuery);
 
+// Right now this only supports a single Mermaid diagram on the page.
+var swimWithTheMermaids = function() {
+  // Are there any diagrams here?
+  if ( 1 > $( '.js-diagrams' ).length ) {
+    return undefined;
+  }
+
+  var diagrams = $( '.js-diagrams' );
+  var padding = 30;
+
+  diagrams.each( function ( idx, el ) {
+    var $el = $( el );
+    var diagramText = $( el ).find( '.js-diagrams__raw' ).text()
+    mermaid.render(
+      $el.find( '.js-diagrams__stage' ).attr( 'id' ),
+      diagramText,
+      function callbackMermaidRender( c ) {
+        $el.find( '.js-diagrams__stage' ).html( c );
+      }
+    );
+    $el.find( '.js-diagrams__stage' ).find( 'style' ).remove();
```

Comment:
Could we do this with css instead of js?

Reply:
it's coming [from the mermaid library here](https://github.com/knsv/mermaid/blob/d6e22d5f5e969d78d6b2e9ab85aaa62119a18b84/src/mermaidAPI.js#L437-L448). so we've got to remove it after the fact for now.